x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -281,32 +283,36 @@ export function TransactionsTable({
       gutterSize="s"
       data-test-subj="transactionsGroupTable"
     >
-      <EuiFlexItem>
-        <EuiFlexGroup justifyContent="spaceBetween" responsive={false}>
-          <EuiFlexItem grow={false}>
-            <EuiTitle size="xs">
-              <h2>
-                {i18n.translate('xpack.apm.transactionsTable.title', {
-                  defaultMessage: 'Transactions',
-                })}
-              </h2>
-            </EuiTitle>
-          </EuiFlexItem>
-          {!hideViewTransactionsLink && (
+      {/* hide the transactions table title for mobile transactions tabs */}
```

Comment:
[nitpick] Maybe we don't need this comment, it will get outdated fast if we try to hide the title for other cases.